Description: Fix several typos
 This patch fixes several typos in the manual of pstoimg
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/345357

--- latex2html-2008-debian1.orig/pstoimg.pin
+++ latex2html-2008-debian1/pstoimg.pin
@@ -446,13 +446,13 @@ banner() unless($opt{quiet});
 
 Use Ghostscript's anti-aliasing feature for rendering "softer" images.
 This applies to lines and edges of polygonal and oval or circular shapes.
-Only valid if Ghostscipt 4.03 or higher is installed.
+Only valid if Ghostscript 4.03 or higher is installed.
 
 =item B<-aaliastext>
 
 Use Ghostscript's anti-aliasing feature for "smoother" font characters,
 without the jagged edges. Similar to B<-antialias> for graphic components.
-Only valid if Ghostscipt 4.03 or higher is installed.
+Only valid if Ghostscript 4.03 or higher is installed.
 
 =item B<-center> I<num>
 
@@ -906,7 +906,7 @@ my $Brx = '-?\d+(?:\.\d*|)';
 =head1 DESCRIPTION
 
 B<pstoimg> iterates over the given input files and runs them through
-Ghostscipt. The resulting pnm (portable anymap files) are processed
+Ghostscript. The resulting pnm (portable anymap files) are processed
 with different Netpbm tools (cropping, color mapping, aligning, ...)
 and finally converted into (currently) either GIF or PNG format. The
 bitmaps can now be included e.g. in WWW pages.

